content/en/logs/guide/restrict-access-to-log-events-with-restriction-queries.md

```diff
@@ -0,0 +1,389 @@
+```
+{{% /tab %}}
+{{% tab "Generic API" %}}
+
+API call:
+
+```
+curl -X POST "https://app.datadoghq.com/api/v2/roles/<ROLE_ID>/permissions" -H "Content-Type: application/json" -H "DD-API-KEY: <DATADOG_API_KEY>" -H "DD-APPLICATION-KEY: <DATADOG_APP_KEY>" -d '{"data": {"type":"type": "permissions","id": <PERMISSION_ID>}}’
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Create Restriction Queries
+
+There are many ways to identify which logs correspond to each team. For example, you can use the service value, or add a `team` tag on your data.
+
+This guide assumes that there is a `team` tag associated with the backend and frontend logs.
+
+{{< tabs >}}
+{{% tab "Backend" %}}
+
+API call:
+```
+curl -X POST "https://app.datadoghq.com/api/v2/logs/config/restriction_queries" -H "Content-Type: application/json" -H "DD-API-KEY: <DATADOG_API_KEY>" -H "DD-APPLICATION-KEY: <DATADOG_APP_KEY>" -d '{"data": {"type": "logs_restriction_queries","attributes": {"restriction_query": "team:backend"}}}'
+```
+
+Response:
+
+```
+{
+	"data": {
+		"type": "logs_restriction_queries",
+		"id": "76b2c0e6-98fa-11ea-93e6-775bd9258d59",
+		"attributes": {
+			"restriction_query": "team:backend",
+			"created_at": "2020-05-18T11:26:48.887750+00:00",
+			"modified_at": "2020-05-18T11:26:48.887750+00:00"
+		}
+	}
+}
+```
+
+{{% /tab %}}
+{{% tab "Frontend" %}}
+
+API call:
+
+```
+curl -X POST "https://app.datadoghq.com/api/v2/logs/config/restriction_queries" -H "Content-Type: application/json" -H "DD-API-KEY: <DATADOG_API_KEY>" -H "DD-APPLICATION-KEY: <DATADOG_APP_KEY>" -d '{"data": {"type": "logs_restriction_queries","attributes": {"restriction_query": "team:frontend"}}}'
+```
+
+Response: 
+
+```
+{
+	"data": {
+		"type": "logs_restriction_queries",
+		"id": "b3228a0c-98fa-11ea-93e6-d30e1d2c52ee",
+		"attributes": {
+			"restriction_query": "team:frontend",
+			"created_at": "2020-05-18T11:28:30.284202+00:00",
+			"modified_at": "2020-05-18T11:28:30.284202+00:00"
+		}
+	}
+}
+```
+
+{{% /tab %}}
+{{% tab "Generic API" %}}
+
+```
+curl -X POST "https://app.datadoghq.com/api/v2/logs/config/restriction_queries" -H "Content-Type: application/json" -H "DD-API-KEY: <DATADOG_API_KEY>" -H "DD-APPLICATION-KEY: <DATADOG_APP_KEY>" -d '{"data": {"type": "logs_restriction_queries","attributes": {"restriction_query": "<QUERY>"}}}'
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+You have now created both the role and the query for the frontend and the backend team. 
+
+Next, restrict access to the backend and frontend by [attaching the restriction query to the created role][9] with the role IDs and query IDs.
+
+## Attach queries to the role
+
+You have the role ID and query ID from the response of the creation call. Use them when attaching the query to the role.
```

Comment:
I think its good to emphasize here that we can only tie 1 query restriction to a role

Reply:
So that is something that I would actually prefer to see in the main documentation rather than hidden in a guide.
Do you think it would make more sense to see it there: https://docs.datadoghq.com/account_management/rbac/permissions?tab=datadogapplication#log-data-access ?

Reply:
Yea agreed.
Would you consider linking to the main doc from the guide?